index.html

```diff
@@ -67,12 +86,12 @@
 
     <main>
       <section>
-        <div class="carousel-container fade">
-          <div class="container fade">
+        <div class="carousel-container">
+          <div class="container">
             <img class="carousel-img" src="/images/carousel-1.jpg" />
           </div>
 
-          <div class="container fade">
+          <div class="container">
             <img class="carousel-img" src="/images/carousel-3.jpg" />
```

Comment:
Add the alt attribute to the image tags.

Reply:
Done. Thanks